prototypes/items.lua

```diff
@@ -240,6 +240,8 @@ data:extend({
   { -- High explosive cannon shell
     type = "ammo",
     name = "high-explosive-cannon-shell",
+    -- added due to 2.0 modding API changes

```

Comment:
We should remove the "added due to 2.0 modding API changes" comments, as the git commit has indicated the changes.